libs/designsystem/src/lib/components/action-sheet/action-sheet.component.html

```diff
@@ -0,0 +1,20 @@
+<button
+  kirby-button
+  [attentionLevel]="attentionLevel"
+  (click)="onToggleSheet($event)"
+  (mousedown)="onButtonMouseEvent($event)"
+  tabindex="-1"
+>
+  <ng-container *ngIf="triggerText">{{ triggerText }}</ng-container>
+  <kirby-icon [name]="triggerIconName"></kirby-icon>
+</button>
+
+<kirby-action-sheet-popout
+  *ngIf="!isTouch"
+  (itemSelect)="onItemSelect($event)"
+  [hideCancel]="true"
+  [disabled]="disabled"
```

Comment:
Shouldn't `disabled` apply to the trigger button ☝🏻  instead? 🤔

Reply:
Indeed. Fixed and cookbook example added